packages/kbn-proc-runner/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@kbn/proc-runner",
+  "main": "./index.js",
```

Comment:
When you build this, as @spalger mentioned, the `"main"` needs to change to the build location, e.g. `./target/index.js`